unicodetools/src/main/java/org/unicode/xml/GeneratePropertyValues.java

```diff
@@ -967,17 +1012,18 @@ private static String getFormattedQuickCheckProperties() {
                         UcdProperty.NFKC_Quick_Check, VALUESOUTPUTTYPE.MAX_LINE_LENGTH)
                 + DOUBLELINE
                 + getFormattedAttribute(
-                        UcdProperty.NFKD_Quick_Check, VALUESOUTPUTTYPE.MAX_LINE_LENGTH)
-                + TRIPLELINE
-                + getFormattedBoolean(UcdProperty.Expands_On_NFC)
-                + DOUBLELINE
-                + getFormattedBoolean(UcdProperty.Expands_On_NFD)
-                + DOUBLELINE
-                + getFormattedBoolean(UcdProperty.Expands_On_NFKC)
-                + DOUBLELINE
-                + getFormattedBoolean(UcdProperty.Expands_On_NFKD)
-                + TRIPLELINE
-                + getFormattedSyntax(UcdProperty.FC_NFKC_Closure);
+                        UcdProperty.NFKD_Quick_Check, VALUESOUTPUTTYPE.MAX_LINE_LENGTH);
+//Deprecated
+//                + TRIPLELINE
+//                + getFormattedBoolean(UcdProperty.Expands_On_NFC)
+//                + DOUBLELINE
+//                + getFormattedBoolean(UcdProperty.Expands_On_NFD)
+//                + DOUBLELINE
+//                + getFormattedBoolean(UcdProperty.Expands_On_NFKC)
+//                + DOUBLELINE
+//                + getFormattedBoolean(UcdProperty.Expands_On_NFKD)
+//                + TRIPLELINE
+//                + getFormattedSyntax(UcdProperty.FC_NFKC_Closure);
```

Comment:
i suggest you just delete this altogether

Reply:
Done